modules/nw-networkpolicy-deny-all-allowed.adoc

```diff
@@ -16,14 +16,12 @@ endif::[]
 [id="nw-networkpolicy-deny-all-multi-network-policy_{context}"]
 = Creating a default deny all {name} policy
 
-This is a fundamental policy, blocking all cross-pod networking other than network traffic allowed by the configuration of other deployed network policies. This procedure enforces a default `deny-by-default` policy.
+This policy blocks _all_ cross-pod networking other than network traffic allowed by the configuration of other deployed network policies and traffic between host-networked pods. This procedure enforces a strong deny policy by applying a `deny-by-default` policy in the `default` namespace.
```

Comment:
@JoeAldinger default namespace is not for users to use it. Users create their own projects/namespaces and apply  network policies to them. In the example deny-by-default is just a name of the policy, it is not at all connected to default namespace.  We should avoid using 'default ' namespace in examples.
It could be **deny-by-default policy in the namespace it is created.**

Reply:
The Jira issue itself is to avoid using default namespace in examples as default namespace is for kubernetes usage.

Reply:
Description
<pre>
NetworkPolicy examples make use of the "default" Namespace. This should be avoided as it is not meant to be used by end-users.
</pre>
Expected results
<pre>
Create a Namespace in current project or inform reader that this is a placeholder and actual target Namespace should be specified. 
</pre>

Reply:
@JoeAldinger We do have to avoid using "default" in examples but use project and namespace created by user for  the workloads.